test/e2e/examples.go

```diff
@@ -167,6 +167,38 @@ var _ = Describe("Examples e2e", func() {
 			})
 		})
 	})
+
+	Describe("[Skipped][Example]Liveness", func() {
+		It("should create and stop liveness pods", func() {
+			mkpath := func(file string) string {
+				return filepath.Join(testContext.RepoRoot, "examples", "liveness", file)
+			}
+			execYaml := mkpath("exec-liveness.yaml")
+			httpYaml := mkpath("http-liveness.yaml")
+			nsFlag := fmt.Sprintf("--namespace=%v", ns)
+
+			runKubectl("create", "-f", execYaml, nsFlag)
+			runKubectl("create", "-f", httpYaml, nsFlag)
+			checkReset := func(podName string, timeout time.Duration) {
+				err := waitForPodRunningInNamespace(c, podName, ns)
+				Expect(err).NotTo(HaveOccurred())
+
+				for t := time.Now(); time.Since(t) < timeout; time.Sleep(poll) {
+					pod, err := c.Pods(ns).Get(podName)
+					expectNoError(err, fmt.Sprintf("getting pod %s", podName))
+					restartCount := api.GetExistingContainerStatus(pod.Status.ContainerStatuses, "liveness").RestartCount
+					Logf("Pod: %s   restart count:%d", podName, restartCount)
+					if restartCount > 0 {
+						return
+					}
+				}
+				Failf("Pod %s was not restarted", podName)
+			}
+			By("Check restarts")
+			checkReset("liveness-exec", 2*time.Minute)
```

Comment:
Can we shrink these timeouts a bit please?  How long does this test typically take to pass?  Also, the second timeout is essentially in addition to the first one.  Can we either do these both in parallel, or reduce the second timeout to zero?